cross/gogs/Makefile

```diff
@@ -0,0 +1,42 @@
+PKG_NAME = gogs
+PKG_VERS = 0.9.13
+PKG_EXT = zip
+PKG_DIST_SITE = https://dl.gogs.io
+PKG_DIR = $(PKG_NAME)
+
+DEPENDS = 
```

Comment:
should depend on `native/go` and build from source (no binary downloading/unpacking if we have source available)